docs/guides/multisig-proposal.md

```diff
@@ -0,0 +1,241 @@
+# Multisig Proposal
+
+After adding FPS to your project dependencies, the next step is to create the first Proposal contract. In this example, we will create a proposal that deploys a new instance of `Vault.sol` and a new ERC20 token, then transfer ownership of both contracts to the multisig wallet.
+
+```solidity
+pragma solidity 0.8.19;
```

Comment:
What do we think about moving these code examples into the `examples/` directory?

Reply:
We have them there, too, IMO we should also keep them in the doc. Otherwise, the dev reading the doc would have to open many files to understand what is going on. What are your thoughts @ElliotFriedman @bsolid ?